cpp/src/slice2rb/RubyUtil.cpp

```diff
@@ -370,17 +370,6 @@ Slice::Ruby::CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
     _out.dec();
     _out << nl << "end"; // End of class.
 
-    //
```

Comment:
All the exact same code is executed by `visitClassDecl` right above this.

And now that we're guaranteed to run `visitClassDecl` before `visitClassDef` (the function you're looking at right now) this code is redundant.